micro-rdk/src/common/conn/viam.rs

```diff
@@ -418,7 +418,7 @@ where
     pub fn run_forever(&mut self) -> ! {
         #[cfg(feature = "esp32")]
         {
-            use esp_idf_svc::sys::{esp_task_wdt_config_t, CONFIG_FREERTOS_NUMBER_OF_CORES};
+            use esp_idf_svc::sys::CONFIG_FREERTOS_NUMBER_OF_CORES;
             let wdt_cfg = crate::esp32::esp_idf_svc::sys::esp_task_wdt_config_t {
```

Comment:
do you min importing `esp_task_wdt_config_t` and removing `crate::esp32::esp_idf_svc::sys::`?